### PR TITLE
Update progress reports terminology

### PR DESCRIPTION
**Pull request by willdch**
[Monday.com Ticket: Update Terminology of Quarterly Reports UI](https://seed-company-squad.monday.com/boards/3451697530/pulses/3830415588)

Made all the requested changes

Screenshot 2023-01-19 at 12 06 18 PM

Screenshot 2023-01-19 at 12 02 11 PM

Screenshot 2023-01-19 at 12 03 16 PM

Screenshot 2023-01-19 at 12 05 22 PM

### CODE REVIEW

File: src/components/PeriodicReports/PeriodicReportsList.tsx [hunk 1]
```diff
@@ -26,7 +26,11 @@ export const PeriodicReportsList = ({
   onRowClick?: (report: PeriodicReportFragment) => void;
   TableCardProps?: CardProps;
 } & StyleProps) => {
-  const reportTypeName = `${type} Reports`;
+  /**
+   * Cheking if reportTypeName is 'Progress' and explicityly setting it to 'Quartlerly'
+   * so we don't need to update the API
+   */
```
Comment (CarsonF) [inline src/components/PeriodicReports/PeriodicReportsList.tsx:32]:
```suggestion
```
Comment just says what the code is doing.

File: src/scenes/ProgressReports/Detail/PromptResponseCard.tsx [hunk 2]
```diff
@@ -42,7 +42,7 @@ export const PromptResponseCard = ({
         <Stack direction="row" justifyContent="space-between">
           {typeof title === 'string' ? (
             <Typography variant="h3" paragraph>
-              {title}
+              {title === 'Community Story' ? 'Story' : title}
```
Comment (CarsonF) [inline src/scenes/ProgressReports/Detail/PromptResponseCard.tsx:45]: Change here instead
https://github.com/SeedCompany/cord-field/blob/f5e48da9cf6307a00fc2ce8b7691ed1b65b30223/src/scenes/ProgressReports/Detail/ProgressReportDetail.tsx#L195-L195

File: src/scenes/ProgressReports/EditForm/ProgressReportStepper.tsx [hunk 3]
```diff
@@ -75,15 +75,19 @@ export const ProgressReportStepper = () => {
...
               {steps.map(([label, StepComp]) => (
                 <Step
                   key={label}
                   onClick={() => setProgressReportStep(label)}
                   active={StepComp === CurrentStep}
                 >
-                  <StepButton icon={' '}>{label}</StepButton>
+                  <StepButton icon={' '}>
+                    {label === 'Community Story' ? 'Story' : label}
+                  </StepButton>
```
Comment (CarsonF) [inline src/scenes/ProgressReports/EditForm/ProgressReportStepper.tsx:90]: Change here instead
https://github.com/SeedCompany/cord-field/blob/f5e48da9cf6307a00fc2ce8b7691ed1b65b30223/src/scenes/ProgressReports/EditForm/Steps/index.ts#L14-L16

**Review by CarsonF — CHANGES_REQUESTED** [review]

File: src/scenes/ProgressReports/EditForm/ProgressReportStepper.tsx (hunk 3 above)
Comment (willdch) [reply src/scenes/ProgressReports/EditForm/ProgressReportStepper.tsx]: Well, I did most of those the hard way.  Making the changes now

**Pull request merged**